charts/redis/templates/redis-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.managed_redis.disabled }}
```

Comment:
Can we rename this file to `Secret.yaml` ?

Reply:
Yup, It's updated